src/ReTestItems.jl

```diff
@@ -388,11 +386,12 @@ function _runtests_in_current_env(
             # Now all workers are started, we can begin processing test items.
             @info "Starting running test items"
             starting = get_starting_testitems(testitems, nworkers)
+            gc_between_testitems = nworkers > 1
```

Comment:
alternatively, we could expose a user-facing keyword to `runtests`, `gc_between_testitems`, which defaults to `nworkers>1`?